### ISSUE TITLE
VASLize board 74 and 75

### ISSUE THREAD

**Issue by styson** [vasl-developers/vasl#272]
WO 2016 gives us 2 more boards for the queue.

**Comment by bkemp01** [comment]
These are in progress.

**Comment by BigAl737** [comment]
Pushing the src files for bd75 as I type. Thanks for your hard work Scott! Doug, ready for your LOS magic when you get time.

**Comment by derimmer** [comment]
Done. 

 

Al, I have updated the src files for bd75 (there was a minor problem with the boardmetadata.xml file plus I bumped the version numbers in the boardmetadata.xml file and the data file) and added the LOSData file to the src/bd75.

 

Attached is the board archive version of bd75. This needs to be added to the vasl-boards-extensions repository and finally it is necessary to update the board version document named “v5boardVersions.txt” file in the VASL code repository /boards directory. I will leave these last two steps in your good hands.

 

Doug

**Comment by styson** [comment]
What is happening with bd74?

**Comment by styson** [comment]
Is board 74 ready for LOS? How does 75 get transitioned to the website?

**Comment by derimmer** [comment]
Hi Sam

 

I have recently updated the LOS on board 75. It is updated in vasl-boards-extensions. 

 

It is still not clear to me how boards updated in vasl-boards-extensions get updated on the vasl.info website. I can’t help with that.

 

I have just recently updated about 40 boards in vasl-boards-extensions as part of the work on VASL6.4.0. They all have new version numbers.  I recommend these not be updated on the website until our vasl6.4.0 beta testing is done. (It will start shortly). They show up in vasl-boards-extensions as updated as part of issue #312.

 

However, board 75 could go on the website as its terrain is not impacted by the los changes. 

 

Doug

**Comment by zgrose** [comment]
> It is still not clear to me how boards updated in vasl-boards-extensions get updated on the vasl.info website. I can’t help with that.

I see this asked a lot. vasl.info is driven off vasl-developers/vasl-website so to upload a new board file (e.g. board 75) you would:

1. create a new branch in vasl-developers/vasl-website (e.g. board-75)
2. add the bd75.zip to the **boards** directory in that branch
3. add board 75 to the versions.json file in that same directory
4. add board 75 to the v6boardsAll.zip file in that same directory
5. create a pull request for your branch to get merged into the main trunk

**Comment by zgrose** [comment]
BTW, are all these new boards ready to go live? Once vasl/boards/v5boardVersions.txt is updated, the auto-updater starts trying to download them...  See http://www.gamesquad.com/forums/index.php?threads/unexpected-behavior-using-ctrl-npad-to-move.129879/#post-1862377

**Comment by derimmer** [comment]
Yes, they are. I have made changes to them as part of 6.4.0. They should still work for people using 6.3.3. 

 

I have not yet put them on the website. I was proposing to do that only once 6.4.0 is officially released. 

 

So they are only available through the auto-updater. I realize that may cause some confusion but I don’t have a better idea. The new boards are necessary to test the changes to LOS that I made in 6.4.0. 

 

The note that I will post on GS/BGG/FB about 6.4.0-beta1 will explain this. 

 

Doug

**Comment by zgrose** [comment]
If you revert the version number changes in v5boardVersions.txt, people should be able to manually install the 6.4 boards while world at large uses 6.3. At the same time you'll be testing the "only download newer boards" patch I put in. :)

**Comment by derimmer** [comment]
I will revert in v5boardVersions.txt just until 6.4.0-beta1 is out. Then I think we want people to get the new boards automatically. 

 

Should get to it tomorrow. 

 

Thanks. 

 

Doug

**Comment by derimmer** [comment]
@zgrose Zoltan

I am following your instructions to update the vasl.info website to (a) add bd75 and (b) update bd19 to delete the recursive file as per #373. All good until I get to 

> 4.add board 75 to the v6boardsAll.zip file in that same directory

at which point I get an error message that the zip file is invalid or corrupted. I can open the other .zip files in the directory. If I download the v6boardsAll.zip file directly from GitHub I get the same error message. 

Could somebody check this? I have no knowledge about how to fix this. The last update to the v6boardsAll.zip file on GitHub was made by @styson on Dec 30, 2016 as far as I can tell.

Thanks.

**Comment by derimmer** [comment]
And by all the boards, you mean 1-75 (minus 74)?

There is no link to this big zip on vasl.info anymore that I can see.

**Comment by zgrose** [comment]
Right, there is no link. But yes, it is a zip of all the boards that exist. Putting back a "gimme all the boards" link probably useful. It was those 01-10, 11-20, etc that seemed like a waste of time.

**Comment by derimmer** [comment]
@zgrose Zoltan. As you can see above, I pushed up some new files to vasl-website today, including the bd75.zip file and the updated .json file. 

Bd75.zip (v6.2) has not shown up on the website yet. Is there something else I need to do? Thanks.

**Comment by zgrose** [comment]
The HTML doesn't write itself for new boards. I can tweak the page for ya and push it up.

**Comment by derimmer** [comment]
OK. Thanks Zoltan. So, 75 is done in vasl/src, vasl-boards-extensions and vasl-website. Zoltan will make it appear on vasl.info.

**Comment by zgrose** [comment]
It's up and I put in a placeholder spot for 74 as well.

**Comment by bkemp01** [comment]
board 74 source files have been added to vasl/boards/src, ready for LOS

**Comment by derimmer** [comment]
Zoltan, I have pushed up bd74.zip to vasl-website and have amended versions.json and boards.htm accordingly and correctly (I think) but no joy on getting board 74 to display. Any thoughts?

**Comment by zgrose** [comment]
Looks good to me...?

<img width="879" alt="screen shot 2017-03-20 at 12 13 00 pm" src="https://cloud.githubusercontent.com/assets/5044847/24117233/9f8f6064-0d66-11e7-9ee8-531eb66420e8.png">

**Comment by derimmer** [comment]
Looks good for me now as well. I guess there was a bit of a lag in there somewhere. Thanks!

**Comment by styson** [comment]
it takes a few minutes for github pages to get updated after a change
